<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FGoodFeaturesToTrackDialog</class>
 <widget class="QDialog" name="FGoodFeaturesToTrackDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblMaxCorners">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Corners</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnMaxCorners">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spnQuality">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblQuality">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Quality</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMinDist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Distance</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spnMinDist">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdOk">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
